### ISSUE TITLE
using scanner master clock output for timing?

### ISSUE THREAD

**Issue by arielletambini** [Brain-Imaging-Center/EEG-fMRI#3]
Hi @BenInglis and all,

I'm returning to getting the EEG-fMRI setup going. One question: we are trying to set up a piece of hardware (a 'syncbox', http://www.brainproducts.com/productdetails.php?id=19 ) which takes in as input the scanner master clock output to synchronize the EEG acquisition relative to the scanner clock. Some more info from the brainvision people (the EEG system that we have) about this is below. [I previously thought that this was the TTL, but apparently not...] Would it be possible to have this signal funneled into the control room via a coax cable (that is what the box takes in as input) - or perhaps this already exists?

from Brainvision people:
"The “scanner master clock” output is usually in a machine room where the MRI hardware and cold head pump are located.  Within the many cabinets in that room, there is a master clock output there which operates in the megahertz range.  This is the clock against which the MRI decides when different steps should be taken for acquiring the MR images.

This is separate from the 5V TTL signal that you mentioned which throws at the start of every TR.  Both signals are important to EEG/fMRI scans for different reasons.

The SyncBox connecting the scanner master clock to the BUA is important for synchronizing the EEG clock with the scanner clock to prevent any phase shifting in the scanner artifacts that are recorded.  By continuously keeping the clocks in sync, the MR gradient artifact is kept very stable improving a researchers ability to cleanly remove the artifact from the signal of interest."

Thanks!
Arielle

**Comment by danshel** [comment]
Hi Arielle

Your link to the "syncbox" info isn't working.

**Comment by arielletambini** [comment]
that is weird... http://www.brainproducts.com/productdetails.php?id=19

maybe try the link from github directly, if you were trying it through email before?

**Comment by danshel** [comment]
Do you already have the syncbox hardware? From one of the diagrams it looks
like the "SyncBox scanner interface" is connected to the scanner master
clock output via a short coax cable. So it looks like this "SyncBox scanner
interface" needs to be located within a few feet of the scanner cabinets.
The "syncbox scanner interface" then connects via a long coax to the
"SyncBox" which I guess would have to be in the scanner operator room.

I would guess that the master clock signal would be in the gradient
cabinet. Can you ask the Brain Products for a confirmation of this and
whether they can give us the name of the port for a Siemens Trio?

**Comment by arielletambini** [comment]
Yup, we already have the syncbox. That layout sounds correct to me (we already have a very long coax cable, hopefully will be long enough). I'll ask them about those bits, thanks!

**Comment by BenInglis** [comment]
Yes I can't find the port name in these documents, that would be useful as Daniel says. We will also need to look at the filtering issue because the coax will need to enter the magnet room at the rear wall. We don't have our own filters there but Siemens may have space for us. I'll email Siemens now and ask if they know the port number we can use (there is probably only one predefined for customer use) and what to do about filtering the coax.

**Comment by RickRedfern** [comment]
We have the port on the scanner located.  The long BNC cable needs to be at
least 50ft.  I'm assuming you also have the BNC to SMA cable that connects
directly to the clock port.

**Comment by BenInglis** [comment]
Here's a photo of the port for the scanner clock, and some description from Siemens:

"CLK10_M2 signal: It gives 3 dBm into a 50 ohm load, which is a bit under one volt p-p.  It is a 10 MHz clock."

![img_3169](https://cloud.githubusercontent.com/assets/6925533/12653956/a724ba9e-c5a6-11e5-8890-c538a53d7a2f.jpg)

With reference to the Brain Products schematic, below, we now need to run the long coax cable over the top of the magnet room, outside the RF shield, to connect between the SyncBox Scanner Interface (which will live on/near the scanner clock signal in the cabinet) and the SyncBox Main Unit (which will live in the operator room). We can't go through the magnet room because we can't (and don't want to) filter the 10 MHz clock signal. We don't need to anyway, since the only connections from the EEG into the magnet room are the two fiber optic cables in the schematic.

Open questions: is the long coax cable long enough to traverse the magnet room? (Rick's last question.) And are the fiber optics long enough to get to the back of the magnet from the operator room, via a wave guide?

Brain Products schematic:

[SyncBox_SetUp.pdf](https://github.com/Brain-Imaging-Center/EEG-fMRI/files/108624/SyncBox_SetUp.pdf)

**Comment by arielletambini** [comment]
Great, thanks everyone! 

Regarding the fiber optic cable, we do have a long enough cable to get from the back of the magnet into the operator room (Rick previously ordered).

Regarding the long coax cable, I think it should be long enough to traverse the magnet room, but we should check (I actually can't any documentation on exactly how long it is, it just says 'long').

Regarding the BNC / SMA connector, I believe that I have that (there are a few connectors that look like this). Rick, could I stop by to check this connector and the coax cable length with you this afternoon (if you are around before 2pm)?

**Comment by RickRedfern** [comment]
Hi Arielle,

I pulled the coax cable from the equipment room to control room today.  I
can't connect up to the scanner yet though.  I've had to order a SMA to SMA
cable for inside the gradient cabinet to go from the clock output to a
filter in the top of the cabinet where your SMA to BNC adapter will
actually connect too.  The gradient cabinet is actually a RF cabinet just
like the scanner room.

Cheers!

**Comment by arielletambini** [comment]
Hi Rick,

Do you happen to know if the SMA/SMA cable for the gradient cabinet has come in yet?

Thanks!
Arielle

**Comment by RickRedfern** [comment]
Hi Arielle,

Went to install it Wednesday and it ends up not to be SMA at all.
Contacted Siemens who will be sending me the correct cable, QLA.  Should be
able to install it next Wednesday morning.

Cheers!
Rick

**Comment by RickRedfern** [comment]
I've just installed the cables and synbox.  You will find the BNC cable in
the control room on the floor between the console and TMS with a yellow
label, EEG FMRI.

Cheers!

**Comment by arielletambini** [comment]
thanks Rick!

**Comment by arielletambini** [comment]
thanks everyone - tested this morning and the connection is working well (the hardware and software are happy)!
